src/atoms/ErrorMessage/index.js

```diff
@@ -9,9 +9,9 @@ import Text from 'atoms/Text';
 import styles from './error_message.module.scss';
 
 const hocErrorMessage = BaseComponent => props =>
-  <TransitionGroup>
+  (<TransitionGroup>
```

Comment:
@Jexeones24 is the linter forcing you to put the `(` on the same line as the jsx? I kinda hate how this looks and think it's silly...but what do i know. would much rather prefer:

```
const Component = (props) => (
  <Child {...props} />
);
```

Reply:
We should follow the linter that's the PG standard. https://github.com/policygenius/athenaeum/pull/592 is the PR open to update Athenaeum to the new js linter

Reply:
but who made this a standard? can we iterate on our standards? not sure why this garbage has to be a thing